server/src/test/java/io/crate/auth/UserAuthenticationMethodTest.java

```diff
@@ -21,45 +21,34 @@
 
 package io.crate.auth;
 
-import io.crate.user.User;
-import io.crate.user.UserLookup;
-import org.elasticsearch.test.ESTestCase;
-import io.crate.user.SecureHash;
-import org.elasticsearch.common.settings.SecureString;
-import org.junit.Test;
+import static org.hamcrest.core.Is.is;
 
-import javax.annotation.Nullable;
 import java.security.NoSuchAlgorithmException;
 import java.security.spec.InvalidKeySpecException;
 import java.util.Collections;
+import java.util.List;
 
-import static org.hamcrest.core.Is.is;
+import org.elasticsearch.common.settings.SecureString;
+import org.elasticsearch.test.ESTestCase;
+import org.junit.Test;
+
+import io.crate.user.SecureHash;
+import io.crate.user.User;
+import io.crate.user.UserLookup;
 
 public class UserAuthenticationMethodTest extends ESTestCase {
 
     class CrateOrNullUserLookup implements UserLookup {
```

Comment:
similar to the prev comment - that's a recently added class and we can drop it by using 
` () -> List.of(User.of("crate", Collections.emptySet(), pwHash)) `

Reply:
I kept it because the pwHash generation requires a try/catch and so on - so it doesn't really help much to turn it into a lambda.